cnf-tests/pod-utils/node/node.go

```diff
@@ -1,12 +1,15 @@
-package pod_utils
+package node
```

Comment:
Can you move node under pod-utils/pkg?

Reply:
Just to make sure, the hierarchy should look like:
```
pod-utils
|_
    pkg
    |_
       node.go
```
and what ```node.go``` should be part of what ```package```?

Reply:
Yes exactly. That's because all the other dirs under pod-utils are binaries